Packages/Ethereum/Sources/SafeAbi/GnosisSafe_v1_0_0.swift

```diff
@@ -0,0 +1,1127 @@
+// THIS FILE IS GENERATED. DO NOT MODIFY BY HAND.
```

Comment:
this class is generated, right? Along with several others in this pr. What were the reasons not to exclude generated code from repository? Could it be excluded and added to .gitignore?

Reply:
yes, but it is not automatically generated and inserted into the build process on every build. I ran the generator manually, so I'm committing this so that we can use these APIs in order to initiate transactions

Reply:
@elgatovital I'd remove it in the future when/if there's a way to auto-generate and insert in the build process